src/components/DeleteConfirmation/DeleteConfirmation.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./DeleteConfirmation.scss";
+
+function DeleteConfirmation(props) {
+  return (
```

Comment:
it's better to destructure props (ie. `{lessonName, handleDelete, handleClose} = props`

Reply:
ill address this in the delete-confirmation branch

Reply:
since i pulled the delete-confirmation branch, its also on this one